docs/advanced_usage.md

```diff
@@ -24,3 +26,31 @@ relayed_packets{chain="cosmoshub-4",channel="channel-141",path="hubosmo",port="t
 relayed_packets{chain="osmosis-1",channel="channel-0",path="hubosmo",port="transfer",type="acknowledge_packet"} 36
 relayed_packets{chain="osmosis-1",channel="channel-0",path="hubosmo",port="transfer",type="recv_packet"} 35
 ```
+
+---
+
+## Auto Update Light Client
+
+By default, the Relayer will automatically update clients (`msgUpdateClient`) if the client has <= 1/3 of its trusting period left. 
+
+> NOTE: The trusting period of the corresponding client is restored with each transaction the Relayer relays. In other words, every time the Relayer relays a message, it also sends a `msgUpdateClient` message restarting the time to the clients expiration. 
+> This auto-update functionality is specifically useful on low trafficked paths where messages aren't regularly being relayed.
+
+
+You can choose to update clients more regularly by using the `--time-threshold` flag when running the `rly start` command.
+
+Example:
+
+- You are relaying on a path that has a client trusting period of 9 minutes.
+- If no messages are sent for 6 minutes and the client is 3 minutes (1/3) to expiration, the relayer will automatically update the client.
+- If you wish to update the client more frequently, say when the client is within 5 min of expiration, use flag: `--time-threshold 5m`
+
+Selecting a time-threshold that is less than 1/3 of the client trusting period will deem itself useless.
+
+> Most chains have a client trusting period of 336h or 504h
```

Comment:
I believe the IBC team recommends a trusting period that is 2/3 of the chains unbonding period

Reply:
That may be the case, but I'm not sure how many chains follow that recommendation. 

I'll remove this line.